src/components/login.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
```

Comment:
There's not much point adding this without looking at auth0, it handles the login form and all sorts for you.